apps/api/src/app/gene/gene.controller.ts

```diff
@@ -0,0 +1,21 @@
+import {
+  Controller,
+  Get,
+} from '@nestjs/common';
+import { GeneService } from './gene.service';
+import {ModuleRef} from "@nestjs/core";
+
+
+@Controller('genes')
+export class GeneController {
+  constructor(private readonly geneService: GeneService) {}
+
+    @Get()
+  async getHugoGenes() {
+    try {
+      return await this.geneService.fetchGeneSymbols();
+    } catch (error) {
+      console.error('Error fetching data:', error);
+    }
```

Comment:
I don't think you should catch the error but rather throw the exception so the frontend is aware.